TruckersFM/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "author": {
+    "name": "Ninja",
+    "id": "150987570715688960"
+  },
+  "service": "TruckersFM",
+  "description": {
+    "en": "TruckersFM is your staiton for driving."
+   },
+  "url": ["truckers.fm"],
```

Comment:
"url": "truckers.fm",

Reply:
not really required as far as i know

Reply:
Yup, not required. I used an array multiple times already.

Reply:
I just put it in because he has to change the other stuff, he might as well change this so it looks better.